templates/index.html

```diff
@@ -1,7 +1,309 @@
+          </div>
+        </div>
+      </div>
+      <div class="col-4 u-equal-height">
+        <div class="p-card">
+          <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/ac9a0e00-lxd_primary.svg" alt="LXD logo">
+          <hr class="u-sv1">
+          <div class="p-card__content">
+            <p>The pure-container hypervisor. Run legacy apps in secure containers for speed and density</p>
+            <p><a href="https://linuxcontainers.org">linuxcontainers.org&nbsp;&rsaquo;</a></p>
+          </div>
+        </div>
+      </div>
+      <div class="col-4 u-equal-height">
+        <div class="p-card">
+          <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/e2ac88af-juju_black-orange_hex.svg" alt="Juju logo">
+          <hr class="u-sv1">
+          <div class="p-card__content">
+            <p>Model-driven cloud-native apps on public and private infrastructure and CAAS</p>
+            <p><a href="https://jaas.ai">jaas.ai&nbsp;&rsaquo;</a></p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row u-equal-height">
+      <div class="col-4 u-equal-height">
+        <div class="p-card">
+          <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/a7916513-picto-openstack.svg" alt="OpenStack logo">
+          &nbsp;
+          <span class="p-heading--four" style="position: relative; top: -7px;">OpenStack</span>
+          <hr class="u-sv1">
+          <div class="p-card__content">
+            <p>Upgrades, maintenance, support, and fully managed options for long-term low-cost infrastructure</p>
+            <p><a href="https://ubuntu.com/openstack">ubuntu.com/openstack&nbsp;&rsaquo;</a></p>
+          </div>
+        </div>
+      </div>
+      <div class="col-4 u-equal-height">
+        <div class="p-card">
+          <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/b81a45e4-kubernetes.svg" alt="Kubernetes logo">
+          &nbsp;
+          <span class="p-heading--four" style="position: relative; top: -7px;">Kubernetes</span>
+          <hr class="u-sv1">
+          <div class="p-card__content">
+            <p>App portability for K8s on VMware, Amazon, Azure, Google, Oracle, IBM and bare metal</p>
+            <p><a href="https://ubuntu.com/kubernetes">ubuntu.com/kubernetes&nbsp;&rsaquo;</a></p>
+          </div>
+        </div>
+      </div>
+      <div class="col-4 u-equal-height">
+        <div class="p-card">
+          <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/02c32f74-snapcraft-logo.png" alt="Snapcraft logo">
+          <hr class="u-sv1">
+          <div class="p-card__content">
+            <p>The app store with secure packages and ultra-reliable updates for multiple Linus distros</p>
+            <p><a href="https://snapcraft.io">snapcraft.io&nbsp;&rsaquo;</a></p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row u-equal-height  u-extra-space">
```

Comment:
Ironically, there is an extra space here before `u-extra-space`

```suggestion
    <div class="row u-equal-height u-extra-space">
```